InstantPrintTool.py

```diff
@@ -277,12 +319,51 @@ def __reloadComposers(self, removed=None):
         self.dialogui.comboBox_composers.blockSignals(False)
         if self.dialogui.comboBox_composers.count() > 0:
             self.dialogui.comboBox_composers.setCurrentIndex(active)
-            self.dialogui.spinBoxScale.setEnabled(True)
+            self.dialogui.comboBox_scale.setEnabled(True)
             self.exportButton.setEnabled(True)
         else:
             self.exportButton.setEnabled(False)
-            self.dialogui.spinBoxScale.setEnabled(False)
+            self.dialogui.comboBox_scale.setEnabled(False)
 
     def __help(self):
         manualPath = os.path.join(os.path.dirname(__file__), "help", "documentation.pdf")
         QDesktopServices.openUrl(QUrl.fromLocalFile(manualPath))
+
+    def scaleFromString(self, scaleText):
+        locale = QLocale()
+        parts = [locale.toInt(part) for part in scaleText.split(":")]
+        try:
+            # catch 0 Division
+            if len(parts) == 2 and parts[0][1] and parts[1][1] and parts[0][0] != 0 and parts[1][0] != 0:
+                return float(parts[0][0]) / float(parts[1][0])
+            else:
+                return None
+        except ZeroDivisionError:
+            return
```

Comment:
Since you catch a zero denominator in the if statement, a ZeroDivisionError exception should never occur.